frontend/src/Components/Table.jsx

```diff
@@ -33,6 +33,15 @@ export default function Table() {
     // ref to keep track of if first Render has happend (this was suggested online)
     const firstRender = useRef(true);
 
+    /*
+        Because the players need to be sitting at arbitary points around the table,
+        absolute positioning is used relative to the table div. Each player is arranged
+        around the table such that they are at an equal angle apart from each other. 
+        The positions of each player (x, y) is calculated using the equations of 
+        an ellipse (as the table is an ellipse). The playerCoordinates are tracked in 
+        the playerCoords state variable. This variable changes is updated when the 
+        window is resized.
+    */

```

Comment:
small nits: 
 arbitary -> arbitrary 
"This variable changes is updated" -> This variable is updated

Reply:
Thank u~